contracts/interfaces/IAxelarAuth.sol

```diff
@@ -2,9 +2,7 @@
 
 pragma solidity ^0.8.9;
```

Comment:
```suggestion
pragma solidity ^0.8.0;
```
change it for other AxelarAuth interface and contract too since we're redeploying those